pipeline/minio/overlays/minioPd/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+nameprefix: minio-
+bases:
+- ../../base
```

Comment:
This might break remote deployments  like this: `kustomize build github.com/kubeflow/manifests/pipeline/...`
Maybe open an issue for this to track?